public/app/features/alerting/unified/utils/rule-form.ts

```diff
@@ -251,34 +283,55 @@ export function rulerRuleToFormValues(ruleWithLocation: RuleWithLocation): RuleF
 
   const defaultFormValues = getDefaultFormValues();
   if (isGrafanaRulesSource(ruleSourceName)) {
-    if (isGrafanaRulerRule(rule)) {
+    // GRAFANA-MANAGED RULES
```

Comment:
Separately, looks like this method could do with some love as its a bit complex/could benefit from some earlier returns/error throwing, I think